UCSLibraries/src/main/java/it/cnr/iit/ucs/requestmanager/RequestManagerInterface.java

```diff
@@ -25,17 +25,16 @@
  * Request manager which will then forwards the request to the context handler.
  *
  * @author Antonio La Marra
- *
  */
-public interface UCSCHInterface {
+public interface RequestManagerInterface {
 
     /**
-     * Sends a message from the external world to the context handler
+     * Receives a message from the external world
      *
      * @param message
      *          the message to be sent
      * @return the response of the ContextHandler
      */
-    public boolean sendMessageToCH( Message message );
+    public boolean sendMessage( Message message );
```

Comment:
when we use requestManager.sendMessage(message) we are not using the rm to send the message right away. We are actually receiving a message to be then stored or forwarded to the contextHandler.
Would it be more logical to call this function requestManager.putMessage or .receiveMessage ?

Reply:
+1
that would make the purpose clearer